documentation/kyverno-cli.md

```diff
@@ -88,5 +88,45 @@ Saving the mutated resource in a file/directory:
 kyverno apply /path/to/policy.yaml --resource /path/to/resource.yaml -o <file path/directory path>
 ```
 
+Apply policy with variables:
+
+Use --set flag to pass the values for variables in a policy while applying on a resource.
+
+```
+kyverno apply /path/to/policy.yaml --resource /path/to/resource.yaml --set <variable1>=<value1>,<variable2>=<value2>
```

Comment:
Can we add a working example here? I'm trying to replace mutated value in [this](samples/best_practices/add_safe_to_evict.yaml) policy: 
```
+(cluster-autoscaler.kubernetes.io/safe-to-evict): variables1
```

with flag `--set=variables1=somevalue`, it did not seem to replace the value., result:
```
kind: Pod
metadata:
  annotations:
    cluster-autoscaler.kubernetes.io/safe-to-evict: variables1
...
```

Reply:
updated readme with example.